playground/entry.js

```diff
@@ -1,14 +1,15 @@
 import { createNomen } from 'nomen-js'
 import { arrowJS } from 'nomen-js/arrow'
 import { preact } from 'nomen-js/preact'
+import { hmr } from 'nomen-js/hmr'
 import routes from './routes.js'
 import { createServer } from '@hattip/adapter-node'
 import { vanilla } from 'nomen-js/vanilla'
 import fs from 'node:fs'
 
 const nomen = createNomen({
   routes: routes,
-  modules: [preact, arrowJS, vanilla],
+  modules: [hmr, preact, arrowJS, vanilla],
```

Comment:
You can just disable `arrowJS` and `preact` modules and use just `vanilla` to make it easier to work with this

Reply:
sure!